app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -53,7 +56,10 @@ public void onCreate(Bundle savedInstanceState) {
 
         setToolbarTitle(getString(R.string.accounts));
 
+        view = findViewById(R.id.drawer);
```

Comment:
I am really shocked that you made this object again, It's already present on top of the file check out `@BindView(R.id.drawer)
    DrawerLayout drawerLayout;`
Please remove this, It's making any sense.